tools/bmh_test/run_local_bmh_test_setup.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash -x
+
+set -xe
+
+# Install virtinst
+sudo apt update
```

Comment:
```suggestion
sudo apt-get -y update
```
`apt` is not meant for scripting, use `apt-get`. You also want to include `-y`, otherwise it might wait for user input.